src/pages/Wishlist/Wishlist.module.css

```diff
@@ -0,0 +1,89 @@
+.products-header {
+  background-image: url("../../assets/images/background/shop-banner.jpg");
+  position: relative;
+  min-height: 60vh;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+.products-header .banner-text {
+  padding-top: 8rem;
+  width: 100%;
+  display: flex;
+  flex-flow: row;
+  justify-content: center;
+  align-items: center;
+  color: var(--COLOR-white);
+  font-size: 2rem;
+}
+
+.products-header .banner-products {
+  padding-top: 1rem;
+  display: flex;
+  flex-flow: row;
+  justify-content: center;
+  gap: 2rem;
+  padding-bottom: 2rem;
+}
+
+.products-header .banner-products a {
+  display: flex;
+  flex-flow: row;
+  justify-content: center;
+  gap: 0.5rem;
+  font-weight: 500;
+  font-size: 1rem;
+  text-decoration: none;
+  color: var(--COLOR-footer-text);
+}
+.banner-products.wishlist{
+    gap: 0.5rem;
+}
+
+.wishlist-main{
+    padding: 2rem;
+    display: flex;
+    flex-flow: column;
+    background-color: #F7F7F7;
```

Comment:
try using variables for colors throughout your application to maintain consistency.